readme.md

```diff
@@ -228,15 +228,14 @@ particular partition. You can do this in two ways.
     ```
 
 #### Sending batches of events
-Sending a batch of messages is more efficient than sending a single message.
+Sending a batch of messages is more efficient than sending a single message. `SendBatch` takes an `*EventBatchIterator` that will automatically create batches from a slice of `*Event`.
 ```go
-batch := &EventBatch{
-            Events: []*eventhub.Event { 
-                eventhub.NewEventFromString("one"),
-                eventhub.NewEventFromString("two"),
-            },
-        }
-err := client.SendBatch(ctx, batch)
+var events []*Event
+events = append(events, NewEventFromString("one"))
```

Comment:
Add package name for public method?  `eventhub.NewEventFromString("one")`, `eventhub.NewEventBatchIterator(events...)`

Reply:
I was keeping it generic, but it does add clarity. If we add the package name i think we should show the import and do it for all objects that would need it.

Reply:
I got your point. But I just want to make sure all the samples in the readme have similar format.

Reply:
added the package name